recipes/oso/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "oso" %}
+{% set build_number = 0 %}
+{% set version = "0.26.1" %}
+{% set min_rust = "rust >=1.46" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/osohq/oso/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 412b62ca46b649cd2b0ddfd9027180eeda9aa8d4f059da44978fa56ac103e90a
+
+build:
+  number: {{ build_number }}
+  script:
+    - bash {{ RECIPE_DIR }}/build_oso.sh
+
+requirements:
+  build:
+    - {{ compiler('c') }}                    # [unix]
+    - {{ compiler('cxx') }}                  # [unix]
+    - {{ compiler('m2w64_c') }}              # [win]
+    - {{ compiler('m2w64_cxx') }}            # [win]
+    - {{ min_rust }}
+  host:
+    - jq
+    - make
+    - clangdev                               # [not osx]
+    - openssl
+    - python
+    - pip
+    - cffi >=1.0.0
+  run:
+    - python
+
+test:
+  source_files:
+    - .
+  imports:
+    - oso
+    - polar
+  commands:
+    - pip check
+    - make python-test
+  requires:
+    - pip
+    - pytest-cov
+    - ruamel_yaml
+    # fails pip check: https://github.com/conda-forge/importlib_metadata-feedstock/issues/88
+    - importlib-metadata !=4.7.1
+    - cffi >=1.0.0
```

Comment:
as `cffi` appears in host, and is needed in `test` to pass `pip check`, it probably belongs in `run` (aside from the `importlib-metadata` snafu). Basically anything in `install_requires` which they load from:

https://github.com/osohq/oso/blob/main/languages/python/oso/requirements.txt#L1